# Offboard point message data

Header header

float32 px_1
float32 py_1
float32 ph_1
float32 px_2
float32 py_2
float32 ph_2
uint8 seq
uint8 total
